src/Kdyby/Redis/ExclusiveLock.php

```diff
@@ -48,13 +53,16 @@ class ExclusiveLock extends Nette\Object
 	public $acquireTimeout = FALSE;
 
 
-
 	/**
 	 * @param RedisClient $redisClient
+	 * @param string|null $keyPrefix
 	 */
-	public function __construct(RedisClient $redisClient)
+	public function __construct(RedisClient $redisClient, $keyPrefix = null)
 	{
 		$this->client = $redisClient;
+		if (!empty($keyPrefix)) {
+			$this->keyPrefix = $keyPrefix . '_';
```

Comment:
I'd preffer `:` as it's prefixing a namespace and you're basically creating a namespace. Also in other places in this library `:` is already used as namespace separator.